kong/plugins/zipkin/opentracing.lua

```diff
@@ -171,18 +171,18 @@ function OpenTracingHandler:log(conf)
 			local try = balancer_tries[i]
 			local span = proxy_span:start_child_span("kong.balancer", try.balancer_start / 1000)
 			span:set_tag(ip_tag(try.ip), try.ip)
-			span:set_tag("peer.port", try.port)
-			span:set_tag("kong.balancer.try", i)
+			span:set_tag("peer.port", tostring(try.port))
+			span:set_tag("kong.balancer.try", tostring(i))
 			if i < balancer_address.try_count then
-				span:set_tag("error", true)
+				span:set_tag("error", "true")
 				span:set_tag("kong.balancer.state", try.state)
-				span:set_tag("kong.balancer.code", try.code)
+				span:set_tag("kong.balancer.code", tostring(try.code))
```

Comment:
The formatting of `tostring` is undefined; could you use `string.format("%d", try.port)` instead? (goes for elsewhere too)